target/linux/ramips/dts/mt7620a_wavlink_wl-wn572hg3.dts

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "wavlink,wl-wn572hg3", "ralink,mt7620a-soc";
+	model = "Wavlink WL-WN572HG3";
+
+	aliases {
+		led-boot = &led_status_high;
+		led-failsafe = &led_status_high;
+		led-running = &led_activity_mid
+		led-upgrade = &led_status_high;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		lan {
```

Comment:
LEDs should be in the form led-0 or led@0 with possibly an additional marking, e.g. led_power: led-0

Reply:
ok. I'll update them